STEPS.md

```diff
@@ -0,0 +1,195 @@
+# How to build Awesome Tasks, step by step
+
+This document describes how to build Awesome Tasks step by step.
+We will be introducing new concepts on each step, so make sure you understand what's going on before going further! If you have any doubts, please ask your coach for help!
+
+## Step 1
+### From basic HTML markup and styling to a dynamic list of tasks
+
+We will start with a new HTML file called `index.html`. We will use it as the skeleton of our little app. Here is how it will look like:
+
+*1.1 —`index.html` base markup*
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Awesome Tasks</title>
+  </head>
+  <body>
+    <h2>Awesome Tasks</h2>
+    <ul id="task-list">
+      <li>Buy coffee</li>
+      <li>Buy milk</li>
+    </ul>
+  </body>
+</html>
+```
+
+If you open this file in a web browser, you will see a pretty boring HTML page that shows a title and a static list of things to do. Not that much, to be honest!
+
+The first thing we are going to do will be taking control of the task list: we will use JavaScript for adding a new task.
+
+Before `</body>`, let’s add this:
+
+*1.2 — The `<script>` tag*
+```html
+<script>
+  var listElement = document.querySelector('#task-list');
```

Comment:
Wouldn't the traditional ```getElementById``` be a little bit easier to grasp for a total newcomer? This would also mean we wouldn't have to explain how the ```#``` and ```.``` prefixes work.